src/routes/scheduleContractsDreSync.js

```diff
@@ -0,0 +1,86 @@
+const { config } = require('../config');
+const { LoggerFactory } = require('warp-contracts');
+const { Queue } = require('bullmq');
+const { getAllContractsIds } = require('../db/nodeDb');
+
+const logger = LoggerFactory.INST.create('scheduleContractsDreSync');
+const isTestInstance = config.env === 'test';
+
+module.exports = async (ctx) => {
+  if (!config.foreignDres || Object.keys(config.foreignDres).length < 1) {
+    logger.info('Cannot sync with remote dres, config missing');
+    ctx.body = 'Foreign DREs sync functionality is disabled';
+    ctx.status = 404;
+    return;
+  }
+
+  const foreignDre = ctx.query.dre;
+  if (!foreignDre || !config.foreignDres[foreignDre]) {
+    logger.info('Cannot sync with remote dre, invalid name');
+    ctx.body = `${foreignDre} not recognized, maybe try: /sync-contracts?dre=dreu&size=2`;
+    ctx.status = 400;
+    return;
+  }
+
+  const { nodeDb } = ctx;
+  const size = ctx.query.size;
```

Comment:
'limit'?

Reply:
Done and renamed endpoint
/contract/initial-registration?dre=dreu&limit=2